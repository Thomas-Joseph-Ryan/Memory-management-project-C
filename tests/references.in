forward x
backward x
type x
set x 1 2 3
forward x
backward x
type x
set y x
set z y x
forward x
forward y
forward z
backward x
backward y
backward z
type x
type y
type z