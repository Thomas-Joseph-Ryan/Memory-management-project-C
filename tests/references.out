> no such key

> no such key

> no such key

> ok

> nil

> nil

> simple

> ok

> ok

> nil

> x

> x, y

> y, z

> z

> nil

> simple

> general

> general

> 
bye
